modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/src/main/resources/site-initializer/fragments/group/src/customer-portal/fragments/information-banner/index.html

```diff
@@ -6,6 +6,7 @@
 
 		<p>
 			${languageUtil.get(locale, "liferays-commercial-offerings-have-been-renamed-from-lxc-to-liferay-saas-lxc-sm-to-liferay-paas-and-dxp-to-liferay-self-hosted", "Liferay's commercial offerings have been renamed from LXC to Liferay SaaS, LXC - SM to Liferay PaaS, and DXP to Liferay Self-Hosted.")}
+			${languageUtil.format(locale, "for-more-information-please-click-x",["<a href='https://help.liferay.com/hc/articles/27114210743437' target='_blank'>" + languageUtil.get(locale, "here", "here") + "</a>"])}
```

Comment:
can we add a space after `"for-more-information-please-click-x",` to stay consistent with the rest of the param commas

Reply:
Formatting-wise...

Samuel... can you also please merge all the commits into one? It keeps the commits simple when we send it to BChan and avoid having us (1) cherry pick multiple commits and (2) possibly reduce merge conflicts as well.